requirements_docker.txt

```diff
@@ -0,0 +1,5 @@
+# pip install -r requirements.txt
```

Comment:
Can you please find a way to have one requirements.txt file so there won't be a need to change in two places?

Reply:
Thanks!  If you are amenable to adding the Gunicorn dependency (the only new one to support Docker) to the main `requirements.txt`, then only one the original one is needed. I wasn't sure if it would be a big deal to others to have a dependency that wasn't used in the main code base. :)

Reply:
It is a medium deal for me :-) 

Perhaps we can use the original requirements.txt and add `pip install gunicorn` in the Dockerfile before/after running `pip install -r requirements.txt`.

I think this keeps us with one requirements.txt file and makes sense as the gunicorn is needed for Docker only.